pypeit/core/extract.py

```diff
@@ -1920,6 +1940,9 @@ def ech_objfind(image, ivar, slitmask, slit_left, slit_righ, order_vec, maskslit
             If True, use the boxcar radius in the skymask
         boxcar_rad: float, optional
             Boxcar radius in arcsec
+        objfindQA_dict: dict, default = None
```

Comment:
Here it may be necessary to pass in the dict.

Reply:
Done. It's a string and the `slitord_id` is appended in the loop over the orders.